src/lib/holocene/icon/svg/svg.svelte

```diff
@@ -11,7 +11,8 @@
   fill="none"
   xmlns="http://www.w3.org/2000/svg"
   class={$$props.class}
+  aria-labelledby="icon-title"
 >
-  <title>{title}</title>
+  <title id="icon-title">{title}</title>
```

Comment:
I'm not 100% certain, but I didn't think `aria-labelledby` is necessary with `<title />` as a child of `<svg />`. Also - we should add `role="img"` to `<svg />` here I think.

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title

check out the Test Cases > Passed Example 1 here https://www.w3.org/WAI/standards-guidelines/act/rules/7d6734/

Reply:
Given we can have many `<Icon />`'s on the same page, having a static ID here may not be the best choice. If we do need an `id` and `aria-labelledby`, we should pass in a unique ID, or maybe generate one from the icon name.

Reply:
R.e. `aria-labelledby`, I've seen both patterns (with and without) so I just went with it to be sure or (as [this resource says](https://www.unimelb.edu.au/accessibility/techniques/accessible-svgs))
> The SVG has an aria-labelledby attribute which references a title element. Note: the title element should be sufficient on it's own, but we added aria-labelledby to be sure.

Good point r.e. unique IDs. I've gone ahead and used the `title` to create the ID, but perhaps we should use something even more unique? I've also added `role="img"`.